utils/helper.js

```diff
@@ -1,14 +1,20 @@
 import axios from "axios";
 import * as dotenv from "dotenv";
+import * as dotenv from "dotenv";
```

Comment:
Can you go through this file and check why there seems to be some lines of the same code appearing twice?